Package.swift

```diff
@@ -97,6 +98,37 @@ func objectServerTestTarget(name: String, sources: [String]) -> Target {
     )
 }
 
+func runCommand() -> String {
```

Comment:
This won't work when actually installing the package as a dependency. Dependency packages are run in a sandbox which doesn't have access to anything (not even the package's own files, which is why we can't read the version from dependencies.list). `swift build` doesn't sandbox the local package being built so it'll work there but not in any other scenario.

Reply:
@tgoyne do you have a tip how we can get this IOPlatformUUID? Or should we not use this?

Reply:
I don't think there's any way to get it that works for any installation methods other than cocoapods.

Reply:
Tested this on Cocoapods and SPM and it is working, tested this on debug and also taking a look at the logs in our metrics app on Mongo. Checking now if works as well in Carthage and XCFramework, I don't know if I'm missing a another use where this will not work

Reply:
We are only using the IOPlatformUUID with SPM and Cocoapods, for other types of installation we will return the Mac address (same as `distinct_id`). This will be helpful in some cases (if a user is using SPM or Cocoapods and is using another SDK), but we don't want to be strict about this info.